File: Vectors , ArrayList, List/Java/RemoveEvenNumber.java
import java.util.ArrayList;

public class RemoveEvenNumbers {
    public static ArrayList<Integer> RemoveEvenNumbers(ArrayList<Integer> v) {
        v.removeIf(val -> val % 2 == 0);
        return v;
    }

    public static void main(String[] args) {
        ArrayList<Integer> v = new ArrayList<>();
        for (int i = 1; i <= 10; i++) {
            v.add(i);
        }

        v = RemoveEvenNumbers(v);
        for (int val : v) {
            System.out.print(val + " ");
        }
        System.out.println();
    }
}

File: Vectors , ArrayList, List/Python/RemoveEvenNumber.py
def remove_even_numbers(v):
    return [val for val in v if val % 2 != 0]

v = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
v = remove_even_numbers(v)
for val in v:
    print(val, end=" ")
print()